crates/sargon-uniffi/src/security_center/support/input.rs

```diff
@@ -0,0 +1,63 @@
+use crate::prelude::*;
+use sargon::CheckSecurityProblemsInput as InternalCheckSecurityProblemsInput;
+use sargon::IsCloudProfileSyncEnabled;
+
+#[derive(Clone, PartialEq, Eq, uniffi::Record)]
+pub struct CheckSecurityProblemsInput {
+    /// Whether the cloud profile sync is enabled.
+    pub is_cloud_profile_sync_enabled: bool,
```

Comment:
You should be able to use InternalConversion if you add From/Into bool for IsCloudProfileSyncEnabled